<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>仿宋</family>
    <pointsize>12</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>网络可靠性</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_SaveAs"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>创建网络</string>
    </property>
    <addaction name="action_UseGuide"/>
    <addaction name="separator"/>
    <addaction name="action_UseGraph"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>计算</string>
    </property>
    <addaction name="action_Minimum"/>
    <addaction name="separator"/>
    <addaction name="action_DMP"/>
    <addaction name="separator"/>
    <addaction name="action_RA"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>查看结果</string>
    </property>
    <addaction name="action_CheckMinimum"/>
    <addaction name="separator"/>
    <addaction name="action_CheckDMP"/>
    <addaction name="separator"/>
    <addaction name="action_CheckRA"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_lineation"/>
   <addaction name="separator"/>
   <addaction name="action_lineation2"/>
   <addaction name="separator"/>
   <addaction name="action_point"/>
   <addaction name="separator"/>
   <addaction name="action_point_2"/>
   <addaction name="separator"/>
   <addaction name="action_turnup"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_UseGuide">
   <property name="text">
    <string>使用向导创建网络</string>
   </property>
  </action>
  <action name="action_UseGraph">
   <property name="text">
    <string>图形化绘制网络</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_Minimum">
   <property name="text">
    <string>最小路集搜索</string>
   </property>
  </action>
  <action name="action_DMP">
   <property name="text">
    <string>D-MP搜索</string>
   </property>
  </action>
  <action name="action_RA">
   <property name="text">
    <string>可靠性评估</string>
   </property>
  </action>
  <action name="action_CheckMinimum">
   <property name="text">
    <string>查看最小路集搜索结果</string>
   </property>
  </action>
  <action name="action_CheckDMP">
   <property name="text">
    <string>查看D-MP搜索结果</string>
   </property>
  </action>
  <action name="action_CheckRA">
   <property name="text">
    <string>查看可靠性评估结果</string>
   </property>
  </action>
  <action name="action_lineation">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/hu/Desktop/画直线.png</normaloff>
     <normalon>../../划线.png</normalon>C:/Users/hu/Desktop/画直线.png</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
   <property name="toolTip">
    <string>绘制直线</string>
   </property>
  </action>
  <action name="action_lineation2">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/hu/Desktop/画曲线.png</normaloff>
     <normalon>../../划线.png</normalon>C:/Users/hu/Desktop/画曲线.png</iconset>
   </property>
   <property name="text">
    <string>曲线</string>
   </property>
   <property name="toolTip">
    <string>绘制曲线</string>
   </property>
  </action>
  <action name="action_point">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/hu/Desktop/画点1.png</normaloff>
     <normalon>../../划线.png</normalon>C:/Users/hu/Desktop/画点1.png</iconset>
   </property>
   <property name="text">
    <string>点</string>
   </property>
   <property name="toolTip">
    <string>绘制点</string>
   </property>
  </action>
  <action name="action_point_2">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/hu/Desktop/橡皮檫1.png</normaloff>
     <normalon>../../划线.png</normalon>C:/Users/hu/Desktop/橡皮檫1.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="action_turnup">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/hu/Desktop/收起工具栏.png</normaloff>
     <normalon>../../划线.png</normalon>C:/Users/hu/Desktop/收起工具栏.png</iconset>
   </property>
   <property name="text">
    <string>收起</string>
   </property>
   <property name="toolTip">
    <string>收起工具栏</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qt.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>showguide()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
